packages/kbn-optimizer/limits.yml

```diff
@@ -27,7 +27,7 @@ pageLoadAssetSize:
   indexLifecycleManagement: 107090
   indexManagement: 140608
   infra: 184320
-  fleet: 100000
+  fleet: 126917
```

Comment:
@spalger yes I followed the steps from the metrics doc, in that PR I am exposing two new components in `public/index ` to be consumed by other plugins it's what is increasing the limit here is there any other patterns we can use to expose react components to other plugins?

Reply:
I guess having a `package-policy-editor` or `fleet-integrations` package that expose components to use for custom integrations could make sense in the future but it's probably not a trivial change . What do you think of increasing the limit for now, and creating an issue to think about moving this to a package? cc @kpollich

Reply:
Moving the Fleet UI extension components to a separate package is an intriguing idea that feels worthwhile. It's a bit painful that they count against Fleet's bundle size limit, but other plugins are able to lazily render them to save on bundle size. We don't have the same luxury since we have to export the components themselves. 

Similar (but tangential) to this is https://github.com/elastic/kibana/issues/110901, which we really need to prioritize based on Fleet's code quality score.

@nchaulet do you mind creating that issue as a follow-up here?

Reply:
Yes sure I just created the issue here https://github.com/elastic/kibana/issues/135078